tools/StaticAnalysis/Exceptions/AzureRM.RecoveryServices.SiteRecovery/BreakingChangeIssues.csv

```diff
@@ -16,3 +16,8 @@
 "Microsoft.Azure.Commands.RecoveryServices.SiteRecovery.dll","Microsoft.Azure.Commands.RecoveryServices.SiteRecovery.GetAzureRmRecoveryServicesAsrStorageClassificationMapping","Get-AzureRmRecoveryServicesAsrStorageClassificationMapping","0","1020","The cmdlet 'Get-AzureRmRecoveryServicesAsrStorageClassificationMapping' no longer has output type 'System.Collections.Generic.IEnumerable`1[Microsoft.Azure.Commands.RecoveryServices.SiteRecovery.ASRStorageClassificationMapping]'.","Make cmdlet 'Get-AzureRmRecoveryServicesAsrStorageClassificationMapping' return type 'System.Collections.Generic.IEnumerable`1[Microsoft.Azure.Commands.RecoveryServices.SiteRecovery.ASRStorageClassificationMapping]'."
 "Microsoft.Azure.Commands.RecoveryServices.SiteRecovery.dll","Microsoft.Azure.Commands.RecoveryServices.SiteRecovery.GetAzureRmRecoveryServicesAsrvCenter","Get-AzureRmRecoveryServicesAsrvCenter","0","1020","The cmdlet 'Get-AzureRmRecoveryServicesAsrvCenter' no longer has output type 'System.Collections.Generic.IEnumerable`1[Microsoft.Azure.Commands.RecoveryServices.SiteRecovery.ASRvCenter]'.","Make cmdlet 'Get-AzureRmRecoveryServicesAsrvCenter' return type 'System.Collections.Generic.IEnumerable`1[Microsoft.Azure.Commands.RecoveryServices.SiteRecovery.ASRvCenter]'."
+"Microsoft.Azure.Commands.RecoveryServices.SiteRecovery.dll","Microsoft.Azure.Commands.RecoveryServices.SiteRecovery.StartAzureRmRecoveryServicesAsrSwitchProcessServerJob","Start-AzureRmRecoveryServicesAsrSwitchProcessServerJob","0","3010","The property 'ChildErrors' of type 'Microsoft.Azure.Management.RecoveryServices.SiteRecovery.Models.HealthError' has been removed.","Add the property 'ChildErrors' back to type 'Microsoft.Azure.Management.RecoveryServices.SiteRecovery.Models.HealthError'."
+"Microsoft.Azure.Commands.RecoveryServices.SiteRecovery.dll","Microsoft.Azure.Commands.RecoveryServices.SiteRecovery.EditAzureRmRecoveryServicesAsrRecoveryPlan","Edit-AzureRmRecoveryServicesAsrRecoveryPlan","0","3030","The generic type for 'property FailoverDirections' has been changed from 'System.Collections.Generic.IList`1[System.Nullable`1[Microsoft.Azure.Management.RecoveryServices.SiteRecovery.Models.PossibleOperationsDirections]]' to 'System.Collections.Generic.IList`1[System.String]'. ","Change the generic type for 'property FailoverDirections' back to 'System.Collections.Generic.IList`1[System.Nullable`1[Microsoft.Azure.Management.RecoveryServices.SiteRecovery.Models.PossibleOperationsDirections]]'."
```

Comment:
Below changes came up because of migration of Enum to String in Swagger spec.
Changes are internally handled by Powershell layer.